docs/usage.md

```diff
@@ -88,6 +95,31 @@ Sample files that can be used as illustrative references can be found in this re
 | last_name | str | User's last name |
 | email | str | User's email |
 
+## `quiz_questions.csv`
+
+| Column | Type | Notes |
+| - | - | - |
+| assessment_id | int | A unique assessment id |
```

Comment:
For consistency with `grades.csv` it might be nice to label this "Assessment ID that can be joined against `assessments.csv`"